content/get-started.yaml

```diff
@@ -0,0 +1,13 @@
+headline: Download our getting started guides in both English and French
+
+body: Download the open source, community version of Graasp Desktop for your preferred operating system.
```

Comment:
Don't we need to change this?

Reply:
I am not using it, will remove

Reply:
OK. For the headline, let's make it shorter and capitalised: "Download Our Getting Started Guide".